monitoring/clientset/versioned/clientset.go

```diff
@@ -7,6 +7,7 @@ import (
 	"net/http"
 
 	monitoringv1 "github.com/openshift/client-go/monitoring/clientset/versioned/typed/monitoring/v1"
+	monitoringv1alpha1 "github.com/openshift/client-go/monitoring/clientset/versioned/typed/monitoring/v1alpha1"
```

Comment:
This set of monitoringv1alpha1 should no longer exist, and Sam removed it in https://github.com/openshift/client-go/commit/ac96784db6c9509b1405312d9cb7f8b2915e5111

However when I do `make generate` it seems to recreate it back. What step am I missing?